deform/widget.py

```diff
@@ -164,14 +165,19 @@ class Widget(object):
         'tinymce' for Tiny MCE).  The ``version_id`` is a string
         indicating the version number (or ``None`` if no particular
         version is required).  For example, a rich text widget might
-        declare ``requirements = (('tinymce', '3.3.8'),)``.  See also:
+        declare ``requirements = (('tinymce', '3.3.8'),)``.
+
+        Requirements specified as a sequence of dicts should be in the form
+        ``({requirement_type: requirement_location(s)}, ...)``.  The
+        ``requirement_type`` key must be either ``js`` or ``css``.  The
+        ``requirement_location(s)`` value must be either a string or a list of
+        strings. Each string must resolve to a concrete resource. For example,
+        a widget might declare ``requirements = ({"js": "deform:static/tinymce/tinymce.min.js"}, {"css": "deform:static/tinymce/tinymce.min.css"}, )``.
```

Comment:
this line is too long, not sure how to format it properly

Reply:
Good catch. Fixed in fd8860a.